src/utils.ts

```diff
@@ -177,3 +177,8 @@ export const parseUrl = (url: string) => {
     return;
   }
 };
+
+export function enumToStringArray<T>(enumObject: T) {
+  const uniqueKeys = new Set(Object.keys(enumObject));
+  return Array.from(uniqueKeys);
```

Comment:
Can be simplified to `return [...new Set(Object.keys(enumObject))]`. Also I think it's a good idea to add comments to util functions with a short description of what they do, seems like this one returns unique keys of an object, but the function name doesn't reflect that.